packages/libs/types/src/PactCommand.ts

```diff
@@ -163,6 +163,10 @@ export type ChainId =
   | '17'
   | '18'
   | '19';
+/**
+ * @alpha
+ */
+export type Type = 'cont' | 'exec';
```

Comment:
Can we move this to the source where it's required? I think it should be in packages/libs/client/src/pact.ts

Ultimately, I want to get rid of @kadean/types altogether. The types should belong to where they're required, and describe an interface for a function or class. Not just be somewhere for general use

Reply:
Fixed this on:
https://github.com/kadena-community/kadena.js/pull/413